src/App/PropertyLinks.cpp

```diff
@@ -2763,6 +2911,49 @@ void PropertyLinkSubList::getLinks(std::vector<App::DocumentObject *> &objs,
     }
 }
 
+void PropertyLinkSubList::getLinksTo(std::vector<App::ObjectIdentifier>& identifiers,
+                                     App::DocumentObject* obj,
+                                     const char* subname,
+                                     bool all) const
+{
+    if (!obj || (!all && _pcScope == LinkScope::Hidden)) {
+        return;
+    }
+    App::SubObjectT objT(obj, subname);
+    auto subObject = objT.getSubObject();
+    auto subElement = objT.getOldElementName();
+
+    int i = -1;
+    for (const auto& o : _lValueList) {
+        ++i;
+        if (o != obj) {
+            continue;
+        }
+        if (!subname || (i < (int)_lSubList.size() && subname == _lSubList[i])) {

```

Comment:
Comments explaining all of I presume edge cases would be really useful. Why this is an edge case and why we handle it like this.